src/mongocxx/client.cpp

```diff
@@ -63,13 +58,13 @@ void client::read_preference(class read_preference rp) {
 }
 
 class read_preference client::read_preference() const {
-    class read_preference rp(bsoncxx::stdx::make_unique<read_preference::impl>(
+    class read_preference rp(stdx::make_unique<read_preference::impl>(
```

Comment:
They are in my other PR too. If that one gets approved first, I'll be sure to rebase it in!